api/specs.proto

```diff
@@ -52,6 +52,12 @@ message NodeSpec {
 	Availability availability = 4;
 }
 
+// NodeAvailability is a wrapper so that specifying availability is optional
+message NodeAvailability {
```

Comment:
But the default of `ACTIVE` is sufficient. We can always patch `gogo` to allow generating a pointer for scalar types but we can't remove this extra message.